easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-2.6.4-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'HarfBuzz'
+version = '2.6.4'
+
+homepage = 'https://www.freedesktop.org/wiki/Software/HarfBuzz'
+description = """HarfBuzz is an OpenType text shaping engine."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+source_urls = ['https://www.freedesktop.org/software/harfbuzz/release/']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['9413b8d96132d699687ef914ebb8c50440efc87b3f775d25856d7ec347c03c12']
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('GObject-Introspection', '1.63.1', '-Python-3.7.4'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('GLib', '2.62.0'),
+    ('ICU', '65.1'),
```

Comment:
Note: this was downgraded to 64.2 in #9842 to avoid version conflicts (see #9841)